### DISCUSSION TITLE
Different Landing page depending on User's location

### DISCUSSION THREAD

**Discussion by ekrocks** [bullhorn/career-portal#433] [Ideas]
I just got back with more requests on different landing pages and I realized I created this ticket a while ago. I had opened a ticket with Bullhorn and they lead me back to the GitHub to ask the technical questions here. 
- Currently our site has different pages for people from different locations (in which they can select manually).
- Afterwards, the user can access the portal site from their respective location in which then the portal will view the careers in that State.

I took a look at ```SettingService``` but I think you pointed at that file for the locale settings. 
Is there any way to save the filters to certain selection such that when user will immediately be shown the jobs in their state?

so if someone clicks "New Brunswick" 
![image](https://user-images.githubusercontent.com/10538258/139087791-270448f9-e701-44a4-8a05-9b392311fdce.png)
from our website, it will set the filters like this?
![image](https://user-images.githubusercontent.com/10538258/139087908-13eef117-be5f-4d1c-9b8c-bb9940047862.png)
when the user clicks on a link to go into the Career Portal?

**Comment by charlesabarnes** [comment]
Hello,  I updated this to the "Idea" Category because this will need to be an additional feature.  There is currently no way to link to a preselected filter as shown in the screenshot.
